src/EA.Weee.Domain/AatfReturn/ReturnScheme.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace EA.Weee.Domain.AatfReturn
+{
+    using System;
+    using System.Collections.Generic;
+    using DataReturns;
+    using Domain.Scheme;
+    using EA.Prsd.Core;
+    using EA.Prsd.Core.Domain;
+
+    public partial class ReturnScheme : Entity
+    {
+        protected ReturnScheme()
+        {
+        }
+
+        public ReturnScheme(Scheme scheme, Return @return)
```

Comment:
You can add some unit tests to ensure that the ReturnScheme catches the null exception when the parameters are supplied as null